src/python/pose_format/bin/directory.py

```diff
@@ -10,11 +8,11 @@
 def find_missing_pose_files(directory: str):
     all_files = os.listdir(directory)
     mp4_files = [f for f in all_files if f.endswith(".mp4")]
-    pose_files = {f.removesuffix(".pose") for f in all_files if f.endswith(".pose")}
+    pose_files = {os.path.splitext(f)[0] for f in all_files if f.endswith(".pose")}
```

Comment:
can we please define and use a function called `removesuffix`? (for example)
```py
def removesuffix(text, suffix):
if text.endswith(suffix):
    return text[:-len(suffix)]
else:
    return text
```

Reply:
sure